<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea_2">
      <property name="styleSheet">
       <string notr="true">#centralWidget{
	background-image: url(:/resources/img/hand-drawn-back-school-background_23-2149464866.jpg);
}</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>578</width>
         <height>446</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="0" column="0">
             <widget class="QScrollArea" name="scrollArea">
              <property name="styleSheet">
               <string notr="true">
#centralwidget{
background-image: url(:/resources/img/hand-drawn-back-school-background_23-2149464866.jpg);}</string>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>532</width>
                 <height>400</height>
                </rect>
               </property>
               <property name="styleSheet">
                <string notr="true">#scrollAreaWidgetContents{
	border-image: url(:/resources/img/loganpage.jpg);
}</string>
               </property>
               <widget class="QLabel" name="label_5">
                <property name="geometry">
                 <rect>
                  <x>50</x>
                  <y>20</y>
                  <width>421</width>
                  <height>51</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-family:'Lucida Calligraphy'; font-size:26pt; font-weight:700;&quot;&gt;Welcome To LWB&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
               <widget class="QLabel" name="label">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>230</y>
                  <width>81</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; text-decoration: underline;&quot;&gt;Username:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
               <widget class="QPushButton" name="Register">
                <property name="geometry">
                 <rect>
                  <x>70</x>
                  <y>370</y>
                  <width>91</width>
                  <height>25</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Register</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_4">
                <property name="geometry">
                 <rect>
                  <x>230</x>
                  <y>100</y>
                  <width>71</width>
                  <height>21</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:700;&quot;&gt;LOG-IN FIRST&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
               <widget class="QPushButton" name="pushButton_login">
                <property name="geometry">
                 <rect>
                  <x>70</x>
                  <y>300</y>
                  <width>91</width>
                  <height>31</height>
                 </rect>
                </property>
                <property name="text">
                 <string>Sign-in</string>
                </property>
               </widget>
               <widget class="QLabel" name="label_3">
                <property name="geometry">
                 <rect>
                  <x>80</x>
                  <y>130</y>
                  <width>371</width>
                  <height>41</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(165, 165, 165);</string>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'Times New Roman'; font-size:14pt; font-weight:700; font-style:italic; color:#7fa0c8;&quot;&gt;Are You Ready To Change Your Life?!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
               <widget class="QLabel" name="label_2">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>270</y>
                  <width>81</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; text-decoration: underline;&quot;&gt;Password:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
               <widget class="QLineEdit" name="lineEdit_username">
                <property name="geometry">
                 <rect>
                  <x>90</x>
                  <y>230</y>
                  <width>121</width>
                  <height>22</height>
                 </rect>
                </property>
               </widget>
               <widget class="QLabel" name="label_6">
                <property name="geometry">
                 <rect>
                  <x>3</x>
                  <y>340</y>
                  <width>201</width>
                  <height>20</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>No Account?? No Issue!!</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
               <widget class="QLineEdit" name="lineEdit_password">
                <property name="geometry">
                 <rect>
                  <x>90</x>
                  <y>270</y>
                  <width>121</width>
                  <height>22</height>
                 </rect>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
               <widget class="QLabel" name="label_pic1">
                <property name="geometry">
                 <rect>
                  <x>220</x>
                  <y>200</y>
                  <width>301</width>
                  <height>211</height>
                 </rect>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <property name="styleSheet">
            <string notr="true">#page_2{
	border-image: url(:/resources/img/loganpage.jpg);

}</string>
           </property>
           <widget class="QLabel" name="r_username">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>50</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>username</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="name_edit">
            <property name="geometry">
             <rect>
              <x>310</x>
              <y>50</y>
              <width>113</width>
              <height>20</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="password_edit">
            <property name="geometry">
             <rect>
              <x>310</x>
              <y>90</y>
              <width>113</width>
              <height>20</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>90</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>password</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>160</x>
              <y>130</y>
              <width>61</width>
              <height>16</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>confirm</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="confirm">
            <property name="geometry">
             <rect>
              <x>310</x>
              <y>130</y>
              <width>113</width>
              <height>20</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
            </property>
           </widget>
           <widget class="QPushButton" name="Submit">
            <property name="geometry">
             <rect>
              <x>230</x>
              <y>170</y>
              <width>80</width>
              <height>18</height>
             </rect>
            </property>
            <property name="text">
             <string>submit</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_12">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>220</y>
              <width>548</width>
              <height>12</height>
             </rect>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
